### PR TITLE
updated recipe/new version for minimock

### PR DESCRIPTION
**Pull request by sarnold**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.
- [x] Ask for feedback from @conda-forge/bioconda-recipes on where this should go/what to do (see: https://github.com/orgs/conda-forge/teams/bioconda-recipes/discussions/9)

### CODE REVIEW

**Comment by sarnold** [comment]
@conda-forge/help-python-c the bio folks don't seem so active (no updates on the discussion thingie).  Can we go ahead and merge this?

File: recipes/minimock/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set name = "MiniMock" %}
+{% set version = "1.2.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sarnold/{{ name|lower }}/archive/{{ version }}.tar.gz
```
Comment (isuruf) [inline recipes/minimock/meta.yaml:9]: Could you use the official repo and add your changes as a patch? Have you asked upstream about letting you have merge rights on their repo?

Comment (sarnold) [reply recipes/minimock/meta.yaml:9]: Upstream is 5 years cold, and I think that's long enough to declare myself "upstream maintainer".  If/when the original author returns I'll be happy to make a PR, but until then I'm all there is.

Comment (isuruf) [reply recipes/minimock/meta.yaml:9]: Looks like upstream was active on github yesterday. Can you open an issue on https://github.com/lowks/minimock/issues?

Comment (sarnold) [reply recipes/minimock/meta.yaml:9]: I can make a PR but the last activity was "Commits on Feb 14, 2016".  What are you seeing that I'm not seeing?

Comment (sarnold) [reply recipes/minimock/meta.yaml:9]: PR pushed.

Comment (sarnold) [reply recipes/minimock/meta.yaml:9]: Hmm, still seeing upstream activity in another repo, but nothing on the minimock PR...  I shall ping upstream again...

Comment (isuruf) [reply recipes/minimock/meta.yaml:9]: Thanks. From what I see from your PR, apart from CI, code formatting, the actual changes needed is a small fix right? Can we just use the upstream repo and apply your PR on top as a patch?

Comment (sarnold) [reply recipes/minimock/meta.yaml:9]: Is there a limit on patch size?  (just checking)

Comment (isuruf) [reply recipes/minimock/meta.yaml:9]: > Is there a limit on patch size? (just checking)

Nope.

Comment (sarnold) [reply recipes/minimock/meta.yaml:9]: aaand we got some attention upstream...

Comment (sarnold) [reply recipes/minimock/meta.yaml:9]: PR is merged, new release on github, nothing on pypi yet.  Can we go ahead with upstream github for now?  I can update again whenever pypi gets updated.

Comment (isuruf) [reply recipes/minimock/meta.yaml:9]: >  Can we go ahead with upstream github for now? I can update again whenever pypi gets updated.

Sure.

Comment (sarnold) [reply recipes/minimock/meta.yaml]: okay! :+1:

**Comment by sarnold** [comment]
@conda-forge/help-python waiting paid off, so this one is now "ready"

**Pull request merged**